### PR TITLE
data lokeshvasnik

### PR DESCRIPTION
**Pull request by lokeshvasnik**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4691"></a>

### CODE REVIEW

File: data/lokeshvasnik.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Lokesh Vasnik",
+    "type": "personal",
+    "bio": "A Front End Developer | Github Lover | Opensource",
+    "avatar": "https://github.com/lokeshvasnik.png",
```
Comment (swagataroy30) [inline data/lokeshvasnik.json:5]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

File: data/lokeshvasnik.json (hunk 1 above)
Comment (lokeshvasnik) [reply data/lokeshvasnik.json:5]: Okay

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by himanshu1221 — APPROVED** [review]
**LGTM:+1:**

**Review by Hrishi5111998 — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**